GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
> how is the git commit appearing in two places in the nuspec a benefit

There are two URLs in the nuspec file that point back to files in this repo: the license and the icon. As it was, these URLs were pointing to the tip of develop, which means if you ever changed the license, all the old packages would suddenly claim the license had changed, which isn't cool. Just like with source server support, what you want is to point at the specific commit that produced it. Now with the commit ID's in the nuspec file, the license and icon always match what was originally intended for that version.

> [the yaml file is] it normally not required

Where do you store the version you want to use then? And was GitVersion simply not being used? IIRC when I built the develop branch without my changes, I got an assembly version of 1.x, IIRC. But the latest nuget package you've released is already 2.x, IIRC. So it seemed to me the versioning story was broken. But is it that I have to manually run GitVersion after a build?